src/test/java/com/michelin/ns4kafka/services/executors/TopicAsyncExecutorTest.java

```diff
@@ -96,18 +96,21 @@ void shouldDeleteTagsAndNotCreateIfEmpty() {
 
         topicAsyncExecutor.alterTags(ns4kafkaTopics, brokerTopics);
 
-        verify(schemaRegistryClient).deleteTag(LOCAL_CLUSTER, CLUSTER_ID_TEST + ":" + TOPIC_NAME, TAG2);
-        verify(schemaRegistryClient).deleteTag(LOCAL_CLUSTER, CLUSTER_ID_TEST + ":" + TOPIC_NAME, TAG3);
+        verify(schemaRegistryClient).dissociateTag(LOCAL_CLUSTER, CLUSTER_ID_TEST + ":" + TOPIC_NAME, TAG2);
+        verify(schemaRegistryClient).dissociateTag(LOCAL_CLUSTER, CLUSTER_ID_TEST + ":" + TOPIC_NAME, TAG3);
     }
 
     @Test
     void shouldCreateTags() {
         Properties properties = new Properties();
         properties.put(CLUSTER_ID, CLUSTER_ID_TEST);
 
-        when(schemaRegistryClient.addTags(anyString(), anyList()))
-            .thenReturn(Mono.empty())
-            .thenReturn(Mono.error(new Exception("error")));
+        when(schemaRegistryClient.associateTags(anyString(), anyList()))
+                .thenReturn(Mono.empty())
+                .thenReturn(Mono.error(new Exception("error")));
+        when(schemaRegistryClient.createTags(anyList(), anyString()))
+                .thenReturn(Mono.just(List.of()))
+                .thenReturn(Mono.error(new Exception("error")));
```

Comment:
The 2nd `.thenReturn` should be removed as well (never triggered)

Reply:
1 other unit test should be added:

- `shouldNotAssociateTagsWhenCreationFails` where `createTags` return a `Mono.error`. Verify that `associateTags` is never called

Hopefully it'll deal with code coverage by the way

Reply:
take in account